db/R/dbfiles.R

```diff
@@ -34,16 +34,17 @@
 dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate, mimetype, formatname, parentid=NA, con, hostname=fqdn()) {
```

Comment:
probably should check the other functions in this file as well, such as posteriors etc.